lib/puppet/type/grafana_ldap_config.rb

```diff
@@ -156,6 +155,7 @@ def generate
   end
 
   def eval_generate
+    require 'toml'
```

Comment:
@joshcooper do you know if it's a good idea to move the require for an external gem into the eval_generate method?

Reply:
@bastelfreak that should be fine, as long as that's the first code path that attempt use use toml. Alternatively, if you implement the [`pre_run_check` lifecycle method](https://github.com/puppetlabs/puppet-specifications/blob/master/language/resource_types.md#pre_run_check) then you could require it there.